doc/api/process.md

```diff
@@ -1382,6 +1382,19 @@ event loop **before** additional I/O is processed.  As a result,
 recursively setting nextTick callbacks will block any I/O from
 happening, just like a `while(true);` loop.
 
+## process.noDeprecation
+<!-- YAML
+added: v0.8.0
+-->
+
+* {boolean}
+
+The `process.noDeprecation` property returns if the `--no-deprecation`
```

Comment:
Possibly change "returns if" to "indicates whether or not"

Reply:
I would go with just "indicates whether", without the "or not".